src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -49,24 +49,26 @@ const SubBudgetCard = ({
 
   const renderCardHeader = (budgetType, budgetId) => {
     const subtitle = (
-      <Stack direction="horizontal" gap={2.5}>
+      <Stack direction="horizontal" gap={2.5} className="pb-4">
```

Comment:
IMO, it's a bit odd to add bottom padding to the subtitle vs. the `Card.Header` component itself, given the spacing we want is more in relation to the containing `Card.Header` vs. the subtitle specifically.

I would recommend moving this to a `Card.Header` class name instead. Also, it looks like the `4` spacer value is not quite matching the default top margin of the `Card.Header`, where there's more whitespace at the top of the `Card` than at the bottom (i.e., `Card.Header` uses `4.5` space value).

I would also recommend conditionally passing this class name for it to return only when necessary (current, non-expired/scheduled budgets):

```jsx
import classNames from 'classnames';

<Card.Header
  className={classNames({
      'mb-4.5': budgetLabel.status !== BUDGET_STATUSES.active,
  })}
  // ...
>
```